cli/awsbatch/utils.py

```diff
@@ -100,3 +100,27 @@ def is_job_array(job):
     :return: true if the job is an array, false otherwise
     """
     return "arrayProperties" in job and "size" in job["arrayProperties"]
+
+
+def is_mnp_job(job):
+    """
+    Check if the given job is an MNP job.
+
+    :param job: the job dictionary returned by AWS Batch api
+    :return: true if the job is mnp, false otherwise
+    """
+    return "nodeProperties" in job and "numNodes" in job["nodeProperties"]
+
+
+def get_job_type(job):
+    """
+    Get the type of the job.
+
+    :param job: the job dictionary returned by AWS Batch api
+    :return: one of ["SIMPLE", "ARRAY", "MNP"]
+    """
+    if is_job_array(job):
+        return "ARRAY"
```

Comment:
I'd like to define something like an "enum" `JobType` and use it here and in the tests like: `get_job_type(job) != JobType.SIMPLE `

Reply:
Unfortunately enums are only supported since Python 3.4: https://www.python.org/dev/peps/pep-0435/